src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -121,8 +121,10 @@ class ItemPage extends React.Component {
       </Link>
       : null;
     const holdings = LibraryItem.getItems(record);
-    const hathiEmbedURL = record.hathiVols && record.hathiVols.length ? `//hdl.handle.net/2027/${record.hathiVols[0].volumeId}?urlappend=%3Bui=embed` : '';
-    const hathiURL = record.hathiVols && record.hathiVols.length ? `https://hdl.handle.net/2027/${record.hathiVols[0].volumeId}` : '';
+    // const hathiEmbedURL = record.hathiVols && record.hathiVols.length ?
```

Comment:
Will this return in the immediate future, or is it better to delete rather than comment-out?